src/stories/LinksSection.stories.jsx

```diff
@@ -0,0 +1,21 @@
+import LinksSection from "../components/LinksSection";
+
+export default {
+  title: "Links Section",
+  component: LinksSection,
+};
+
+export const Default = {
```

Comment:
i noticed the behavior of the section starts getting iffy when you have a lot of links or have a link with a really long title. the two concerns might not necessarily be realized since we know generally what to expect for the links currently but could be worth accounting for that in the test cases here and component design above